### PR TITLE
Created Review Fixtures with review instances.

### PR DESCRIPTION
**Pull request by rohanpreetam**
Created the fixtures with a single review and three reviews with the fields:

- id
- Student user id (just the number in the users table)
- itemId 
- Date item was served
- Status (initally "Awaiting Moderation")
- userid of moderator if approved/rejected (initially null)
- Moderator comments (initally null)
- CreatedDate
- LastEditedDate

**Purpose**

This helps us create instances of the review entity which will be used in future testing.

### CODE REVIEW

**Review by li-kelly — DISMISSED** [review]
LGTM

**Review by li-kelly — DISMISSED** [review]
LGTM

File: frontend/src/fixtures/reviewFixtures.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    dateEdited: "2022-01-01T12:00:00",
+  },
+  threeReviews: [
+    {
+      id: 1,
+      studentId: 4921081,
+      itemId: "A02341",
+      dateItemServed: "2022-01-02T12:00:00",
+      status: "Approved",
+      userIdModerator: "moderator1",
```
Comment (pconrad) [inline frontend/src/fixtures/reviewFixtures.js:20]: userIdModerator will be an integer, not a string.

File: frontend/src/fixtures/reviewFixtures.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      dateCreated: "2022-01-01T12:00:00",
+      dateEdited: "2022-01-02T13:00:00",
+    },
+    {
+      id: 2,
+      studentId: 4921082,
+      itemId: "B01321",
+      dateItemServed: "2022-02-02T12:00:00",
+      status: "Rejected",
+      userIdModerator: "moderator2",
```
[the remark this answers is not in the document]
Comment (rohanpreetam) [reply frontend/src/fixtures/reviewFixtures.js]: Modified the file according to the new requirements. Thanks!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**